reference/tools/cmake/cmake.rst

```diff
@@ -1,12 +1,65 @@
 .. _conan-cmake-build-helper:
 
 CMake
------
+=====
+
+The ``CMake`` build helper is a wrapper around the command line invocation of cmake. It will abstract the
+calls like ``cmake --build . --config Release`` into Python method calls. It will also add the argument
+``-DCMAKE_TOOLCHAIN_FILE=conan_toolchain.cmake`` (from the generator ``CMakeToolchain``) to the ``configure()`` call.
+
+The helper is intended to be used in the ``build()`` method, to call CMake commands automatically
+when a package is being built directly by Conan (create, install)
+
+
+.. code-block:: python
+
+    from conans import ConanFile
+    from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps
+
+    class App(ConanFile):
+        settings = "os", "arch", "compiler", "build_type"
+        requires = "hello/0.1"
+        options = {"shared": [True, False], "fPIC": [True, False]}
+        default_options = {"shared": False, "fPIC": True}
+
+        def generate(self):
+            tc = CMakeToolchain(self)
+            tc.generate()
+            deps = CMakeDeps(self)
+            deps.generate()
+
+        def build(self):
+            cmake = CMake(self)
+            cmake.configure()
+            cmake.build()
+
+
+.. note::
+
+    This helper includes the additional flag `-DCMAKE_SH="CMAKE_SH-NOTFOUND"` when using the `MinGW Makefiles` CMake's
```

Comment:
Not anymore, this helper takes this CMAKE_SH variable from the CMakePresets.json file now